BLUESPAWN-client/headers/user/bluespawn.h

```diff
@@ -47,4 +47,4 @@
 
 void print_help(cxxopts::ParseResult result, cxxopts::Options options);
 void dispatch_hunt(cxxopts::ParseResult result, cxxopts::Options options);
-void dispatch_mitigations_analysis(cxxopts::ParseResult result, cxxopts::Options options);
\ No newline at end of file
+void dispatch_mitigations_analysis(cxxopts::ParseResult result, cxxopts::Options options, IOBase& io);
```

Comment:
I'd prefer to eventually see Bluespawn be a class with an iobase member that it could pass around to things. For now, this works

Reply:
this sounds good